Simulator/tests/unit/components.js

```diff
@@ -338,4 +341,47 @@ describe("Components", function() {
       gatherDataFrame.should.be.ok();
     });
   });
+
+  describe("StandardConnections", function() {
+    it("Should Render", function () {
+      standardConnections = shallow(
+        <StandardConnections
+          alias="alias"
+          tables=[{id: "id1", alias: "alias1"},
+                  {id: "id2", alias: "alias2"}]
+          joins=[{left: {tableAlias: "alias1", columnId: "c1"},
+                  right: {tableAlias: "alias2", columnId: "c2"},
+                  joinType: "inner"}]
+        />
+      );
+      standardConnections.should.be.ok();
+    });
+  });
+
+  describe("StandardConnectionValidator", function() {
+    it("Should Render", function() {
+      validator = shallow(
+        <Validator
+          errors=["error1", "error2"]
+        />
+      );
+      validator.should.be.ok();
+    });
+  });
+
+  describe("JoinViz", function() {
+    it("Should Renger", function() {
+      joinViz = shallow(
+        <JoinViz
+          alias="alias"
+          tables=[{id: "id1", alias: "alias1"},
+                  {id: "id2", alias: "alias2"}]
+          joins=[{left: {tableAlias: "alias1", columnId: "c1"},
+                  right: {tableAlias: "alias2", columnId: "c2"},
+                  joinType: "inner"}]
+        />
+      );
+      joinViz.should.be.ok();
+    })
```

Comment:
I think these could use fleshing out. I would look at the tablePreview tests as an example of testing more complicated components with multiple interactions and member functions